roles/sos-vhostuser/files/vhostuser

```diff
@@ -11,6 +11,14 @@ fi
 source /etc/vhostuser-bind.conf
 
 NW_DATA="/var/config/openstack/latest/network_data.json"
+if [ ! -f ${NW_DATA} ]; then
+    echo "Network data file not found, nothing to do"
```

Comment:
remove the "nothing to do" :) because you are going to do something, try to download it.

Reply:
Is there a known reason why the config drive will not work moving forward with 4.10?

Reply:
I'm experimenting on a cloud where I don't enable config-drive to the workers.
What kind of issues do you think I'll run into?

This is purely for my curiosity. BTW with that patch I was able to enable the driver on the interface.